### PR TITLE
Create a data model

### PR DESCRIPTION
**Pull request by Diegogagan2587**
# Implemented changes
- Generate migration files to create tables
   - Include User table
   - Include Comment table
   - Include Like table
   - Include Post table

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,245 @@
...
+  npx stylelint "**/*.{css,scss}" --fix
+  ```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (MahmoudBakr23) [inline README.md:243]: - [x] Please fix the MIT link for a professional README file

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @Diegogagan2587 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done creating the correct migration files with the correct data for each table model
- Well done adding foreign keys and references
- Descriptive PR
- Correct Linters

However, there is always room for improvement.

> I'd like to remind you that your long hours of work are appreciated. However, the code review process is for improving your code, not criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

N/A

Cheers, and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Diegogagan2587) [reply README.md:243]: @MahmoudBakr23 Thanks for pointing that out! it's fixed now!

**Review by Reem-lab — APPROVED** [review]
# Hi @Diegogagan2587,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [x]  Followed Gitflow 💯
- [x]  Descriptive PR⭐
- [x]  Kudos for working on the schema🤩
- [x]  No Linters error ✅



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**